tests/InputFiles/Cce/AnalyticTestBouncingBlackHole.yaml

```diff
@@ -0,0 +1,44 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: AnalyticTestCharacteristicExtract
+# Check: parse;execute_check_output
+# Timeout: 10
+# OutputFileChecks:
+#   - Label: "check_news"
+#     Subfile: "/News.dat"
+#     FileGlob: "CharacteristicExtractVolume*.h5"
+#     ExpectedDataSubfile: "/News_expected.dat"
```

Comment:
I dislike outputting only the error because it does not provide information about the scale of the expected solution, and therefore prevents usefully checking relative error. It seems like a reasonable practice if you know ahead of time that the solution is O(1), but that's not guaranteed for this case.

Reply:
You actually only use absolute tolerances in the input files currently, so should those perhaps switch to relative tolerances then? I'm actually not sure what's the right thing to do here. Your point on relative comparisons applies to evolution and elliptic errors just the same. I feel like it's more useful in practice to output errors, since that's what you typically want to plot and take norms of, plus it seems silly to output the same analytic data at every timestep / iteration. If you say we don't care that the CCE does this differently than the evolution and elliptic code then that would be fine with me. If we want to make it consistent though, then you could output errors here and keep your absolute tolerances in the input file unchanged. When writing tolerances in input files the author has to be aware of the scales and the expected accuracy of the solution anyway, don't they? What do you think?

Reply:
One important note here is that most of the CCE analytic solutions are not static solutions -- I see the point for elliptic solves, and for evolutions comparing to a static solution, but here it is valuable to know what the solution is actually doing when something happens with the errors.
I guess the elliptic solve and the evolution would handle this primarily by constructing the volume output as well when applicable, or adding some reduction data. 
However, given that the diagnostic of output data from the system is useful, then I don't see much of a philosophical difference in outputting the solution and errors to expected solution vs the solution and the expected solution directly.
When writing tolerances you need information about different things for absolute vs relative. For absolute you need the scale of the solution, which can vary depending on how complicated the analytic solution is -- we shouldn't anticipate that it's static in the evolution case. For relative tolerance, it depends on the convergence order and precision of the method, which I think should typically be more available.

Reply:
ok, I think this is ultimately your decision as the author of the code.